modules/social_features/social_profile/config/modify/social_profile.taxonomy_permissions.yml

```diff
@@ -0,0 +1,35 @@
+dependencies:
+  module:
+    - social_registration_fields
+    - taxonomy_access_fix
+items:
+  user.role.anonymous:
+    expected_config:
+      permissions: { }
+    update_actions:
+      add:
+        permissions:
+          - 'view terms in interests'
+          - 'view term names in interests'
+          - 'select terms in interests'
+          - 'view terms in expertise'
+          - 'view term names in expertise'
+          - 'select terms in expertise'
+          - 'view terms in profile_tag'
+          - 'view term names in profile_tag'
+          - 'select terms in profile_tag'
+  user.role.authenticated:
```

Comment:
I think this one is actually not depending on `social_registration_fields` as authenticated user (and verified and such) you should be able to view and select these, as they should be able to edit their profile.

Reply:
The edit profile page is working without this permission, but the registration page using registration fields modules are looking for the new permission.